app/src/sharedTest/java/org/oppia/android/app/databinding/StateAssemblerMarginBindingAdaptersTestActivityTest.kt

```diff
@@ -117,44 +116,44 @@ class StateAssemblerMarginBindingAdaptersTest {
   fun testMarginBindingAdapters_explorationViewMargin_ltrIsEnabled_marginsForLtrIsCorrect() {
     val textView = activityRule.scenario.runWithActivity {
       val textView: TextView = it.findViewById(R.id.test_margin_text_view)
-      setExplorationSplitViewMargin(
+      setExplorationViewMargin(
         textView,
         /* isApplicable= */ true,
-        /* marginStart= */ convertToPx(24f),
-        /* marginTop= */ convertToPx(24f),
-        /* marginEnd= */ convertToPx(40f),
-        /* marginBottom= */ convertToPx(24f)
+        /* marginStart= */ 24f,
+        /* marginTop= */ 24f,
+        /* marginEnd= */ 40f,
+        /* marginBottom= */ 24f
       )
       return@runWithActivity textView
     }
     assertThat(textView.marginStart.toFloat()).isWithin(TOLERANCE).of(
-      convertToPx(24f)
+      /* marginStart= */ 24f
     )
     assertThat(textView.marginEnd.toFloat()).isWithin(TOLERANCE).of(
-      convertToPx(40f)
+      /* marginEnd= */ 40f
     )
   }
 
   @Test
   fun testMarginBindingAdapters_explorationViewMargin_rtlIsEnabled_marginsForRtlIsCorrect() {
     val textView = activityRule.scenario.runWithActivity {
       val textView: TextView = it.findViewById(R.id.test_margin_text_view)
-      setExplorationSplitViewMargin(
+      ViewCompat.setLayoutDirection(textView, ViewCompat.LAYOUT_DIRECTION_RTL)
+      setExplorationViewMargin(
         textView,
         /* isApplicable= */ true,
-        /* marginStart= */ convertToPx(24f),
-        /* marginTop= */ convertToPx(24f),
-        /* marginEnd= */ convertToPx(40f),
-        /* marginBottom= */ convertToPx(24f)
+        /* marginStart= */ 24f,
+        /* marginTop= */ 24f,
+        /* marginEnd= */ 40f,
+        /* marginBottom= */ 24f
       )
-      textView.layoutDirection = View.LAYOUT_DIRECTION_RTL
       return@runWithActivity textView
     }
-    assertThat(textView.marginEnd.toFloat()).isWithin(TOLERANCE).of(
-      convertToPx(24f)
-    )
     assertThat(textView.marginStart.toFloat()).isWithin(TOLERANCE).of(
-      convertToPx(40f)
+      /* marginStart= */ 24f
```

Comment:
Using argument comments doesn't quite make sense in this context, I think. The comment always corresponds to the argument name; ``of()`` doesn't have an argument with name ``marginStart``. Beyond that, the purpose of this comment is to provide context for literals. This assertion line has sufficient context since it's evaluating ``marginStart``. Please remove the argument comment here.

Ditto for all other assertion lines in this & all other test suites in this PR.

Reply:
Thanks for clarification. Done.